ops/pebble.py

```diff
@@ -12,9 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Client for the Pebble API (HTTP over Unix socket).
+"""Client for the Pebble API.
 
-For a command-line interface for local testing, see test/pebble_cli.py.
+This module provides:
```

Comment:
It provides more than what's listed here, so we need something like:

```suggestion
This module provides a way to interact with Pebble, including:
```

Reply:
done